packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-validation.spec.ts

```diff
@@ -114,258 +113,266 @@ describe('DbasS API Error Handling', () => {
-
-        //  Select a region from the dropdown.
-        ui.regionSelect.find().click().type(`${region}{enter}`);
-
-        // Select a resource (Database Clusters) from the autocomplete input.
-        ui.autocomplete
-          .findByLabel('Database Clusters')
-          .should('be.visible')
-          .type(`${clusterName}{enter}`)
-          .click();
-        cy.findByText(clusterName).should('be.visible');
-
-        // Select a Node from the autocomplete input.
-        ui.autocomplete
-          .findByLabel('Node Type')
-          .should('be.visible')
-          .type(`${nodeType}{enter}`);
-
-        // Wait for the metric definitions API call to resolve.
-        cy.wait('@getMetricDefinitions');
-        cy.get('[data-qa-error-msg="true"]')
-          .should('be.visible')
-          .and('have.text', 'Error loading metric definitions');
-      }
-    );
-
-    it( 'should return ' + statusCode +' error response when fetching Services API Request',() => {
-        cy.intercept('GET', apiMatcher(`/monitor/services`), {
-          statusCode: statusCode,
-          body: {
-            errors: [
-              {
-                reason: 'Bad Request',
-              },
-            ],
-          },
-        }).as('fetchServices');
-        cy.visitWithLogin('monitor/cloudpulse');
-
-        // Wait for the API call to complete and capture the response.
-        cy.wait('@fetchServices');
-
-        cy.get('[data-qa-textfield-error-text="Dashboard"]')
-          .should('be.visible')
-          .invoke('text')
-          .then((text) => {
-            expect(text).to.equal('Failed to fetch the services');
-          });
-      }
-    );
-
-    it( 'should return ' + statusCode + ' error response when fetching Dashboards API Request',() => {
-        mockGetCloudPulseServices(serviceType).as('fetchServices');
-        cy.intercept(
-          'GET',
-          apiMatcher(`/monitor/services/${serviceType}/dashboards`),
-          {
-            statusCode: statusCode,
-            body: {
-              errors: [
-                {
-                  reason: 'Bad Request',
-                },
-              ],
-            },
-          }
-        ).as('fetchDashboard');
-
-        cy.visitWithLogin('monitor/cloudpulse');
-        //  Wait for both the fetch services and fetch dashboard API calls to complete.
-        cy.wait(['@fetchServices', '@fetchDashboard']);
-
-        // Assert that the error message for fetching the dashboards is displayed correctly.
-        cy.get('[data-qa-textfield-error-text="Dashboard"]')
-          .should('be.visible')
-          .invoke('text')
-          .then((text) => {
-            expect(text).to.equal('Failed to fetch the dashboards');
-          });
+  it('should return error response when fetching metric definitions API request', () => {
+    cy.intercept(
+      'GET',
+      apiMatcher(`/monitor/services/${serviceType}/metric-definitions`),
+      {
+        statusCode: 400,
```

Comment:
@agorthi-akamai , eventhough 400 is good that it is thrown by API, we can have 500 as default for now, as it indicates there is some real failure in API rather UI sending a bad request